tests/preconversion_assessment/test_generate_report_message.py

```diff
@@ -0,0 +1,9 @@
+from scripts.preconversion_assessment_script import generate_report_message, STATUS_CODE
+
+
+def test_generate_report_message():
+    """All possible status codes result in string message"""
+    for status_code in STATUS_CODE:
+        message = generate_report_message(status_code)
+        assert isinstance(message, str)
+        assert message != ""
```

Comment:
Maybe we could compare it against some output instead of just checking if that is not an empty string?

Maybe we could change it to a parametrize and assert the message based on the level.

Reply:
Alright, parametrized, check the expected strings, I was kinda suprised SKIP is worse than WARNING - but maybe it's correct